### PR TITLE
Yarn monster

### PR DESCRIPTION
**Pull request by bootcamp-Mo**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: node app.js
```
Comment (bootcamp-Mo) [inline Procfile:1]: for heroku

File: server/config/connection.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
+require('dotenv').config()
 const mongoose = require('mongoose');
 
 
-const connectionString = process.env.MONGODB_URI || 'mongodb://127.0.0.1:27017/social-network-api';
-
-mongoose.connect(connectionString).catch(error => console.error('Error connecting to MongoDB', error))
+mongoose.connect(process.env.MONGO_URL, console.log(`Connected to MongoDB Atlas`) || 'mongodb://127.0.0.1:27017/brainsync', console.log(`Connected to MongoDB Atlas`)).catch(error => console.error('Error connecting to MongoDB', error))
```
Comment (bootcamp-Mo) [inline server/config/connection.js:5]: changing database path to go thought mongodb server database(?)

File: server/models/comment.js [hunk 3]
```diff
@@ -1,12 +1,9 @@
 const { Schema, model } = require('mongoose');
+
 const dateFormat = require('../utils/dateFormat');
 
 const commentSchema = new Schema(
 	{
-		commentId: {
```
Comment (bootcamp-Mo) [inline server/models/comment.js:6]: I think this is still needed but I dont know with nested comments so I'll look in to it

File: server/models/post.js [hunk 4]
```diff
@@ -21,12 +21,15 @@ const postSchema = new Schema(
 			default: Date.now,
 			get: (timestamp) => dateFormat(timestamp),
 		},
-		postAuthor: {
+		username: {
```
Comment (bootcamp-Mo) [inline server/models/post.js:24]: I changed it from author post to username for my clarity we can change it back if needed

**Review by BrianMorse1 — APPROVED** [review]
These all make sense to me!

**Pull request merged**